payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt

```diff
@@ -296,6 +304,25 @@ class GooglePayPaymentMethodLauncher @AssistedInject internal constructor(
             get() = merchantCountryCode.equals(Locale.JAPAN.country, ignoreCase = true)
     }
 
+    @Parcelize
+    data class ShippingAddressConfig @JvmOverloads constructor(
```

Comment:
Is there an API review for this change?

Reply:
I left a comment in the ticket https://jira.corp.stripe.com/browse/RUN_MOBILESDK-2395